src/website/index.html

```diff
@@ -2,17 +2,17 @@
 <html lang="en">
```

Comment:
The template should be named `index.njk`; this will enable code highlighting.

Reply:
renamed, do i have to create another pull request for this?

Reply:
@undefined-moe There are multiple issues; it is recommended to debug locally, for example, the icons in the HTML are not previewing.

You're welcome to submit a new PR.